node/packages/parser/src/ethereum/utils.ts

```diff
@@ -17,3 +18,6 @@ export const toNetworkType = (network: Network): NetworkTypes => {
       throw new Error('unsupported network')
   }
 }
+
+const itemToFragment = (item: ABI.Item) => ({ ...item, gas: item.gas?.toString() })
```

Comment:
Assuming this is all `abi-decoder` massaging logic?

Reply:
Yep, but it's all now gone because https://github.com/shapeshift/unchained/pull/267/commits/5606fc19cb7ae325866e20a8fa0d124f4330d5c0 swaps out `abi-decoder` with `ethers` 👌